core/src/main/java/org/elasticsearch/action/ingest/delete/DeletePipelineAction.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.elasticsearch.plugin.ingest.transport.delete;
+package org.elasticsearch.action.ingest.delete;
```

Comment:
we can, I just followed the convention that we currently have in core. I can move all the actions to the same package, not sure everything should be under org.elasticsearch.ingest though, maybe transport actions do deserve their own package.